docs/ops.html

```diff
@@ -3923,6 +3923,10 @@ <h5 class="anchor-heading"><a id="kraft_reconfig_add" class="anchor-link"></a><a
 
   After starting the controller, the replication to the new controller can be monitored using the <code>bin/kafka-metadata-quorum.sh describe --replication</code> command. Once the new controller has caught up to the active controller, it can be added to the cluster using the <code>bin/kafka-metadata-quorum.sh add-controller</code> command.
 
+  The <code>--command-config</code> needs to point to the to-be-added controller's configuration file.
+
+  The <code>controller.quorum.bootstrap.servers</code> property in <code>--command-config</code> file needs to contain an active controller in the quorum.
+
```

Comment:
can you also address the first bullet point in kafka-18149 while you're at it? 
> point to correct location for sample controller configs (e.g. config/kraft/controller.properties)

I just re-confirmed with a downloaded 3.9 distribution that example controller properties file is in a different location
`config/controller.properties ` -> `config/kraft/controller.properties`